meta/autoid/autoid.go

```diff
@@ -46,6 +46,7 @@ const (
 	PerformanceSchemaDBID int64 = SystemSchemaIDFlag | 10000
 	// MetricSchemaDBID is the metrics_schema schema id, it's exported for test.
 	MetricSchemaDBID int64 = SystemSchemaIDFlag | 20000
+	SysSchemaDBID    int64 = SystemSchemaIDFlag | 40000
```

Comment:
Please add some comments. Does the number `40000` is a magic number, just need to be different from others?

Reply:
OK.

> Does the number 40000 is a magic number, just need to be different from others?
Yes.